specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/apigateway.json

```
@@ -666,6 +666,29 @@
           "type": "string",
           "description": "ETag of the resource.",
           "readOnly": true
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The kind of gateway.",
+          "enum": [
+            "KubernetesManaged",
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "GatewayKind",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "KubernetesManaged",
+                "description": "Gateway managed by Kubernetes resources."
+              },
+              {
+                "value": "Default",
+                "description": "Default kind of gateway."
```

Let's clarify how it is managed
```suggestion
                "description": "Kind of gateway that is capable of exposing APIs for workspaces using configuration managed through Azure Resource Manager (ARM)."
```

---

Done.